frontends/web/src/routes/account/send/components/confirm/confirm.module.css

```diff
@@ -0,0 +1,86 @@
+.bitBoxContainer {
+  margin-bottom: var(--space-half);
+}
+
+.bitBoxContainer img {
+  max-width: 220px;
+}
+
+.confirmationItemWrapper {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.confirmItem {
+  margin-bottom: calc(var(--space-half) + var(--space-quarter));
+}
+
+.confirmItem p {
+  margin: 0;
+  word-break: break-all;
+}
+
+.confirmItem ul {
+  list-style-type: square;
+  margin: 0;
+  margin-top: var(--space-quarter);
+  padding-left: var(--space-half);
+}
```

Comment:
Lmk if there's already a way we use square bullet points somewhere - or if we wanna style it a certain way (it's pretty "plain" for now).